modules/dailymotionBidAdapter.js

```diff
@@ -90,6 +92,20 @@ function getVideoMetadata(bidRequest, bidderRequest) {
   return videoMetadata;
 }
 
+/**
+ * Check if user sync is enabled for Dailymotion
+ *
+ * @return boolean True if user sync is enabled
+ */
+function isUserSyncEnabled() {
+  const syncEnabled = deepAccess(config.getConfig('userSync'), 'syncEnabled');
+
```

Comment:
Can be simplified with `if (!syncEnabled) return false;` before test with iframe and pixel